api/python/t4/packages.py

```diff
@@ -402,6 +405,8 @@ def update(self, new_keys_dict, meta=None):
         pkg = self._clone()
         for logical_key, entry in new_keys_dict.items():
             pkg = pkg.set(logical_key, entry, meta)
+        # modified package should have new top hash
+        pkg._unset_tophash()
```

Comment:
Isn't this unset by the `pkg.set` call above?

Reply:
it sure is, will fix